<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PartGui::TaskCFFlange</class>
 <widget class="QWidget" name="PartGui::TaskCFFlange">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>125</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Conflat flange</string>
  </property>
  <property name="sizeGripEnabled" stdset="0">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Parameter</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QGridLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>6</number>
        </property>

				<item row="0" column="0">
         <widget class="QLabel" name="textLabel1">
          <property name="text">
           <string>CF</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="Gui::QuantitySpinBox" name="cf">
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
          <property name="unit" stdset="0">
           <string notr="true">mm</string>
          </property>
          <property name="minimum">
           <double>16</double>
          </property>
          <property name="value">
           <double>400</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="textLabel2">
          <property name="text">
           <string>Pipe diameter</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="Gui::QuantitySpinBox" name="pipeDiameter">
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
          <property name="unit" stdset="0">
           <string notr="true">mm</string>
          </property>
          <property name="minimum">
           <double>-1</double>
          </property>
          <property name="value">
           <double>400</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="textLabel3">
          <property name="text">
           <string>Pipe wall width</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="Gui::QuantitySpinBox" name="pipeWallWidth">
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
          <property name="unit" stdset="0">
           <string notr="true">mm</string>
          </property>
          <property name="minimum">
           <double>0</double>
          </property>
          <property name="value">
           <double>400</double>
          </property>
         </widget>
        </item>
        </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Gui::QuantitySpinBox</class>
   <extends>QWidget</extends>
   <header>Gui/QuantitySpinBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tubeOuterRadius</tabstop>
  <tabstop>tubeInnerRadius</tabstop>
  <tabstop>tubeHeight</tabstop>
 </tabstops>
 <resources>
  <include location="../Gui/Resources/Part.qrc"/>
 </resources>
 <connections/>
</ui>
